test/data_test.go

```diff
@@ -33,13 +34,13 @@ port: 80
 headers:
 User-Agent: "ModSecurity CRS 3 Tests"
 Host: "localhost"
-Content-Type: "application/x-www-form-urlencoded"
+Content-Type: header_values.ApplicationXWwwFormUrlencoded
```

Comment:
Does this require a string substitution? I mean, is this part of the template? E.g.
```suggestion
Content-Type: {{ header_values.ApplicationXWwwFormUrlencoded }}
```
?